### DISCUSSION TITLE
Ideias, created by Daniel 👩‍💻

### DISCUSSION THREAD

**Discussion by ghost** [wezlley126/Starfood#8] [Ideas]
Minhas ideias para o site são:

**Search bar**- Vai servir para deixar mais dinâmico e facilitar a procura por parte dos usuários. Poderíamos usar o **input** com o **type "text"** e **placeholder** como **Search ou Pesquise..**

Tipo isso aqui: 
https://www.w3schools.com/howto/howto_css_searchbar.asp

Exemplos: 

![images.jpeg](https://user-images.githubusercontent.com/90204204/132528399-30de1b80-3511-4219-b41d-0cff85cb2e53.jpeg)

![images (1).jpeg](https://user-images.githubusercontent.com/90204204/132528414-bb77d13c-07de-4956-800e-7a39adae8d31.jpeg)

————————————————————————

**User page** - Seria a página do usuário, onde ele poderia mudar o nickname, a foto de perfil, o email etc. 

**_Foto de Exemplo (obs: Perfil aleatório)_**

![Gallery_1631112973341.png](https://user-images.githubusercontent.com/90204204/132533768-61fe4c77-3353-4fee-aabc-b89fbe8d5bb3.png)

![Gallery_1631113007992.png](https://user-images.githubusercontent.com/90204204/132533786-0c615eae-0bca-410d-8dc4-23a52aedd38d.png)

————————————————————————

**Grid-layout** - Seria um layout mais responsivo para o nosso site. Foi o layout que o Lucas deu ideia de fazer.

![images (2).jpeg](https://user-images.githubusercontent.com/90204204/132534781-bf9e7a21-5880-4f7a-a1c7-95a3210035d2.jpeg)

————————————————————————

**Reduzir os códigos** - É uma dica minha, a gente reduzir os códigos e deixar eles mais organizados colocando comentários e tals, pelo menos não tem chance de nós nos perdermos no meio dos códigos e depois esquecer o que fez.

![dribbble-code.gif](https://user-images.githubusercontent.com/90204204/132537547-de0fbae2-26b0-428c-9b5c-e4c7501e2346.gif)

————————————————————————
**Carrinho de Compras:** Seria uma forma da pessoa comprar várias coisas ao msm tempo. Por exemplo, o Guilherme quer pedir uma pizza, dois hambúrguer e um sushi, ele não precisar comprar e pedir um de cada vez, confirmar a compra e tals. Ele pode pedir tudo de uma vez, aí o preço é calculado etc. (Obs: essa ideia foi sugerida pelo Wesley)

————————————————————————

**Sistema de avaliação:**  Seria um sistema onde os clientes poderiam avaliar as comidas e colocar um comentário seja ele positivo ou não. Teria estrelas, tipo de 1 até 5 ou 6 estrelas, aí as pessoas poderiam colocar 3 estrelas, 4 ou 5 (tanto faz) e o comentário delas...e com isso apareceria lá nas informações de cada comida a média das notas que essas comidas tem, por exemplo: 

O Big Hambúrguer tem as notas dadas por clientes:  4, 5, 3, e 2. Logo, a sua média será :

4 + 5 + 3 + 2 / 4 = 14/4 = 3,5...

3,5 seria a média das avaliações dessa comida..

(Obs: Essa ideia foi sugerida pela Duda) 

**Exemplos (Obs: Da Play Store):**

![Gallery_1631215895665.png](https://user-images.githubusercontent.com/90204204/132750981-43152070-e990-4919-a3f6-a4d6bdb6b1ba.png)

![Gallery_1631215909014.png](https://user-images.githubusercontent.com/90204204/132751003-4ead55f6-41fb-4c76-8a08-08c76dd9e8ca.png)

![Gallery_1631215921305.png](https://user-images.githubusercontent.com/90204204/132751025-85af3a05-96f1-4db2-a111-d1e5210e527f.png)

**Comment by lucasdelima255** [comment]
Cara isso poder da certo tudo e uma questao de estudar

**Reply by ghost** [reply]
Sim sksmskkssksk

**Comment by WeslleyBorges1267** [comment]
Com relação a barra de pesquisa ainda não sei se da pra fazer só com php sem o mysql, mas vamos ver semana que vem.

**Reply by ghost** [reply]
Mano, n entendo muito de back-end e nem de Data Base mas tipo, acho que precisa meio que armazenar os nomes de cada comida e quando a pessoa digitar algo próximo ao nome da comida, vai aparecer sugestões...exemplo:

A pessoa digita **Hamb**

Aí vai aparecer como sugestão, todos os nomes de hambúrguer..tendeu?kskssksks...

**Reply by WeslleyBorges1267** [reply]
eu sei fazer isso com mysql, mas não sei se também existe essa função no php o Lucas disse que tem, então deve ter, ele diz depois como que faz lá.

**Reply by ghost** [reply]
Kksksksk pser, se a gente conseguir fazer isso, vai ser muito daora

**Comment by ghost** [comment]
Essa parte de avaliações eu acho que podemos fazer com php, js, css e html... já essa parte dos carrinhos e da barra de pesquisa podemos fazer com SQL

**Reply by WeslleyBorges1267** [reply]
No caso se o gilmario não deixar colocar mysql agora eu vou colocar um dia depois que acabasse essas provas.

**Reply by ghost** [reply]
Mano, eu acho que na proxima aula, ele vai dar mais um pouco de SQL e dps vai parar de dar aula para a gente ter tempo de discutir sobre o site e tals. Então, eu acho que ele vai deixar ksksskksksk

**Reply by ghost** [reply]
Mas, se ele não deixar...ai a gente pode ver o que podemos adicionar só usando php msm ksks
